### PR TITLE
Upgrade to the last Karma version and upgrade the Node version

### PR DESCRIPTION
**Pull request by steniobhz**

### CODE REVIEW

**Review by mcweba — APPROVED** [review]
We wait with merging until we got feedback from @hiddenalpha because we don't use the hook-js module.

**Comment by hiddenalpha** [comment]
> We wait with merging until we got feedback from @hiddenalpha because we don't use the hook-js module.

I did ping co-workers which are able to judge "*karma/npm"* stuff.

File: gateleen-delegate/src/main/java/org/swisspush/gateleen/delegate/DelegateHandler.java [hunk 1]
```diff
@@ -33,21 +33,21 @@
...
+ "headers" : [],
+ "uri": "/gateleen/server/copy",
+ "method": "POST",
+ "payload": {
+ "source": "/gateleen/$1?expand=100&amp;zip=true",
+ "destination": "/gateleen/zips/users/$1.zip"
+ }
+ }
+ ]
+ }
```
Comment (hiddenalpha) [inline gateleen-delegate/src/main/java/org/swisspush/gateleen/delegate/DelegateHandler.java:50]: ??

Comment (mcweba) [reply gateleen-delegate/src/main/java/org/swisspush/gateleen/delegate/DelegateHandler.java]: These were commits for another PR. They are removed again.

**Comment by mcweba** [comment]
> > We wait with merging until we got feedback from @hiddenalpha because we don't use the hook-js module.
> 
> I did ping co-workers which are able to judge "_karma/npm"_ stuff.

Can you ping them again? :-)

**Pull request merged**